.idea/runConfigurations/Generate_Parser.xml

```diff
@@ -10,11 +10,11 @@
       </option>
       <option name="taskNames">
         <list>
-          <option value=":generateRustParser" />
+          <option value=":generateRustLexerAndParser" />
```

Comment:
Maybe it's better to add `generateRustLexer` task into `taskNames` list to launch both gradle tasks instead of new task creation?

Reply:
:+1: Done